README.rst

```diff
@@ -136,6 +136,12 @@ We will only create Async operators for the "sync-version" of operators that do
 For example, we won’t create an async Operator for a ``BigQueryCreateEmptyTableOperator`` but will create one
 for ``BigQueryInsertJobOperator`` that actually runs queries and can take hours in the worst case for task completion.
 
+To create a async operators we need to inherit from airflow sync operators or
+if sync version isn't available then from airflow ``BaseOperator``.
+
+To create a async sensors we need to inherit from sync sensors or
+if sync version isn't available then from airflow ``BaseSenseOperator``.
```

Comment:
Did we have a discussion on whether Async sensors should inherit from BaseOperator or BaseSensorOperator ? 
I read somewhere @dstandish mentioning that async sensors should inherit from BaseOperator

Reply:
Yes i had a discussion with @kaxil  during the meetup, he suggested moving all sensors under BaseSenseOperator and also creating a ticket for this particular issue which you mentioned in the backlog and then deciding whether it's BaseOpertaor vs BaseSensorOperator?

Reply:
Created a ticket for the discussion on the same[ here](https://app.zenhub.com/workspaces/airflow-providers-team-61a89ee7fbf048001dc8378d/issues/astronomer/astronomer-providers/558).